### ISSUE TITLE
bug: Stripe plugin doesn't get any of the line items

### ISSUE THREAD

**Issue by Evaldash** [cloudquery/cloudquery#10392]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When syncing data from Stripe to MSSQL (or a file), for the tables stripe_payment_links, and stripe_checkout_sessions the field line_items is completely empty.

### Expected Behavior

line_items should be populated with products in JSON format.

### CloudQuery (redacted) config

```
kind: source
# Common source-plugin configuration
spec:
  name: stripe
  path: cloudquery/stripe
  version: "v1.2.0"
  tables: ["*"]
  destinations: ["mssql"]
  skip_tables:
    - stripe_issuing*  # Needs sign-up at https://stripe.com/issuing
    - stripe_treasury* # Needs sign-up at https://stripe.com/treasury
    - stripe_sigma_scheduled_query_runs # Live keys only
    
    
    #- stripe_accounts
    #- stripe_application_fees
    #- stripe_billing_portal_configurations
    #- stripe_capabilities
    #- stripe_charges
    #- stripe_coupons
    - stripe_credit_notes
    #- stripe_customers
    #- stripe_fee_refunds
    #- stripe_file_links
    #- stripe_files
    - stripe_identity_verification_reports
    #- stripe_invoice_items
    #- stripe_invoices
    #- stripe_payment_intents
    #- stripe_payment_methods
    #- stripe_plans
    #- stripe_promotion_codes
    #- stripe_quotes
    - stripe_radar_early_fraud_warnings
    #- stripe_refunds
    - stripe_reporting_report_runs
    - stripe_reviews
    #- stripe_shipping_rates
    - stripe_subscription_schedules
    - stripe_subscriptions
    #- stripe_tax_rates
    #- stripe_terminal_locations
    #- stripe_terminal_readers
    #- stripe_topups
    #- stripe_transfers
    - stripe_webhook_endpoints

  #backend: local
  # Stripe specific configuration
  spec:
    api_key: "secret"
    stripe_debug: true
```

### Steps To Reproduce

1. Follow the How-to: https://www.cloudquery.io/integrations/stripe/mssql , populating the API keys and logins as instructed

### CloudQuery (redacted) logs

[cloudquery.log](https://github.com/cloudquery/cloudquery/files/11351157/cloudquery.log)

### CloudQuery version

Cloudquery version 2.5.3

### Pull request (optional)

- [ ] I can submit a pull request

**Comment by disq** [comment]
Thanks for reporting, will take a look.

**Comment by disq** [comment]
@Evaldash Looks like we need to make separate API calls per checkout session and payment link to be able to get the line items. I've upgraded the Stripe SDK and added the necessary tables (`stripe_checkout_session_line_items` and `stripe_payment_link_line_items`, respectively). They will be available in the next release (once it goes through our code review process).

**Comment by disq** [comment]
@Evaldash We've released [Stripe v1.3.0](https://www.cloudquery.io/docs/plugins/sources/stripe/overview) which adds two new tables mentioned above and should hopefully solve your issue. Feel free to reopen this issue or create a new one if the new version doesn't work for you or if there are other complications.
